crates/client/src/lib.rs

```diff
@@ -629,6 +629,11 @@ impl<R: RegistryStorage, C: ContentStorage, N: NamespaceMapStorage> Client<R, C,
                     packages: Cow::Borrowed(&last_known),
                 })
                 .await
+                .inspect(|res| {
+                    for warning in res.warnings.iter() {
+                        tracing::warn!("WARNING: {}", warning.message);
```

Comment:
Nit: `WARNING:` here is probably unnecessary; `tracing` subscribers will typically annotate with the event level already.

It would probably be more helpful to describe the _type_ of warning, e.g.
```suggestion
                        tracing::warn!("Fetch warning from registry: {}", warning.message);
```

Reply:
Yeah, that tracks as far as I'm concerned. 👍🏻